tox.ini

```diff
@@ -9,7 +9,7 @@ install_command = pip install -c {env:UPPER_CONSTRAINTS_FILE:https://git.opensta
 setenv = VIRTUAL_ENV={envdir}
          PYTHONWARNINGS=default::DeprecationWarning
 deps = -r{toxinidir}/test-requirements.txt
-       -egit+https://git.openstack.org/openstack/neutron.git@stable/newton#egg=neutron
+       -egit+https://github.com/sapcc/neutron.git@stable/mitaka-m3#egg=neutron
```

Comment:
I would suggest to make it overridable with an environment variable, like the upper-constraints.

Reply:
I'll take care of it